tests/channels/clone_channel.ts

```diff
@@ -0,0 +1,48 @@
+import { tempData } from "../ws/start_bot.ts";
+import { assertEquals, assertExists } from "../deps.ts";
+import { cache } from "../../src/cache.ts";
+import { cloneChannel } from "../../src/helpers/channels/clone_channel.ts";
+import { delayUntil } from "../util/delay_until.ts";
+
+Deno.test({
+  name: "[channel] clone a channel",
+  async fn() {
+    const cloned = await cloneChannel(tempData.channelId, "testing");
```

Comment:
I would prefer you create a channel with bitrate & topic etc. and clone this channel

Reply:
You can't create a channel with a bitrate and topic. Topic would require text channel bitrate would be voice.

I would recommend every type of channel being cloned with each property

Reply:
Reworked clone tests (see tests/channels/clone_channel.ts)